### PR TITLE
NXP-28802: Make fixed part of document history pp configurable (10.10)

### PR DESCRIPTION
**Pull request by michaelva**

### CODE REVIEW

**Comment by troger** [comment]
T&P: https://qa2.nuxeo.org/jenkins/job/TestAndPush/job/ondemand-testandpush-troger-10.10/51/

File: nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-audit/src/test/java/org/nuxeo/elasticsearch/TestESHistoryProvider.java [hunk 1]
```diff
@@ -429,6 +435,29 @@ public void testCustomDocumentHistoryPageProvider() {
...
+        searchDoc.setPropertyValue("basicauditsearch:eventCategories", null);
+        searchDoc.setPropertyValue("basicauditsearch:startDate", null);
+        searchDoc.setPropertyValue("basicauditsearch:endDate", null);
+
+        //test with doc
+        PageProvider<LogEntry> pageProvider = getPageProvider("FIXED_PART_DOCUMENT_HISTORY_PROVIDER", 30, 0, doc);
+        pageProvider.setSearchDocumentModel(searchDoc);
+        assertEquals(1, pageProvider.getCurrentPage().size());
+
+        //test with proxy to check that the doc uuid is correctly set in the fixed part
```
Comment (efge) [inline nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-audit/src/test/java/org/nuxeo/elasticsearch/TestESHistoryProvider.java:455]: This comment and the one above has the correct space after `//` on master.

Comment (michaelva) [reply nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-audit/src/test/java/org/nuxeo/elasticsearch/TestESHistoryProvider.java]: Thanks for catching it.

**Comment by michaelva** [comment]
@kevinleturc do you mean squashed?

**Comment by kevinleturc** [comment]
This is the same thing, squash keeps the commit message whereas fixup doesn't.

You can use one or the other. Thanks

**Pull request merged**